src/coreComponents/physicsSolvers/fluidFlow/wells/WellSolverBase.hpp

```diff
@@ -233,6 +233,22 @@ class WellSolverBase : public SolverBase
                                           CRSMatrixView< real64, globalIndex const > const & localMatrix,
                                           arrayView1d< real64 > const & localRhs ) = 0;
 
+  /**
+   * @brief apply a special treatment to the wells that are shut
+   * @param time_n the time at the previous converged time step
+   * @param dt the time step size
+   * @param domain the physical domain object
+   * @param dofManager degree-of-freedom manager associated with the linear system
+   * @param matrix the system matrix
+   * @param rhs the system right-hand side vector
+   */
+  virtual void applyWellShutDown( real64 const time_n,
```

Comment:
Is the `apply` in `applyWellShutDown` mandatory of could it be `shutDownWell`?

Reply:
Done, I changed these function names to `shutDownWell`, I agree it is better